index.html

```diff
@@ -0,0 +1,150 @@
+	  	    </span>			
+  		</li>
+  		<li>
+	  		<i class="fas fa-database fa-3x"></i>
+	  		<p>Fundamentals of Accelerated Data Science</p>
+	  	    <span>
+	  	     	RAPIDS is a collection of data science libraries that allows end-to-end GPU acceleration for data science workflows.
+	  	    </span>			
+  		</li>
+  		<li>
+	  		<i class="fas fa-stream fa-3x"></i>
+	  		<p>Fundamentals of Accelerated Computing with CUDA C/C++</p>
+	  	    <span>
+	  	     	The CUDA computing platform enables the acceleration of CPU-only applications to run on the world’s fastest massively parallel GPUs.
+	  	    </span>			
+  		</li> 
+  		<li>
+	  		<i class="fab fa-connectdevelop fa-3x"></i>
+	  		<p>Fundamentals of Deep Learning</p>
+	  	    <span>
+	  	     	Discover how deep learning works through hands-on exercises in computer vision and natural language processing. You’ll train deep learning models from scratch, learning tools and tricks to achieve highly accurate results. You’ll also learn to leverage freely available, state-of-the-art pre-trained models to save time and get your deep learning application up and running quickly.
+	  	    </span>			
+  		</li> 		  		
+  	</ul>
+  	<a rel="noopener noreferrer" href="https://www.nvidia.com/gtc/training/#tabcomponentenhanced290f9bb7_5748_4e21_9a4a_b3a7c9ff75a2-tab-e232" 
+  	   target="_blank" id="program-btn">
+  	      REGISTER FOR GTC
+  	</a>
+
+  	<a 
+  	rel="noopener noreferrer" href="https://www.nvidia.com/gtc/training/#tabcomponentenhanced290f9bb7_5748_4e21_9a4a_b3a7c9ff75a2-tab-e232" 
+  	target="_blank" id="replace-btn">SEE THE WHOLE PROGRAM</a>		
+  </section>
+  <section id="speakers">
+  	
+  </section>
+  <footer>
+	 <section id="partners">
+	 	<h2>Some of Sponsors</h2>
+	 	<hr>
+	 	  <ul>
+		 	  <li><img alt="partner1" src="./images/general/partner1.svg"></li>
+		 	  <li><img alt="partner2" src="./images/general/partner2.svg"></li>
+		 	  <li><img alt="partner3" src="./images/general/partner3.svg"></li>
+		 	  <li><img alt="partner4" src="./images/general/partner4.svg"></li>
+		 	  <li><img alt="partner5" src="./images/general/partner5.svg"></li>
+		 	  <li><img alt="partner6" src="./images/general/partner6.svg"></li>
+		 	  <li><img alt="partner7" src="./images/general/partner7.svg"></li>
+		 	  <li><img alt="partner8" src="./images/general/partner8.svg"></li> 	
+	    </ul>
+	    <a rel="noopener noreferrer" href="https://www.nvidia.com/gtc/sponsors/" 
+	       target="_blank">link to all the sponsors</a>
+	 </section>
+	 <section id="copyright">
+	 	<img alt="local logo" src="./images/general/local-logo.png">
+
+	 	<p>
+	 		<a rel="noopener noreferrer" target="_blank" href="https://twitter.com/search?q=%23GTC22&amp;src=typd">#GTC22</a>
+	 	  USA - United States
+	 	</p>
+	 </section>     	 
+  </footer>	
```

Comment:
- [ ] On the home page, for the mobile, the footer is not displayed. Kindly fix it.
- [ ] Kindly adjust the layout of the footer to match that of the given design. check out the screenshot and compare the logo and the texts besides.
- Your design 
![Screenshot from 2022-02-04 02-59-59](https://user-images.githubusercontent.com/20245164/152449361-fb34b400-6c7e-4a3f-bdf7-d93dddfc792f.png)

- Given design
![Screenshot from 2022-02-04 02-55-16](https://user-images.githubusercontent.com/20245164/152449281-0050969b-d93a-4600-a875-9d5daa9501a2.png)

- Kindly do the same for the mobile view.

Reply:
- [ ] Hello please make sure to implement the reviewer's suggestion. **You have the latest commit that states you've fixed this although the commit does not have any changes. please kindly make sure to make this change and request a third review. Thanks.

Reply:
- [ ] Hello please make sure to implement the reviewer's suggestion https://github.com/saied2035/capstone-project1/pull/1#discussion_r799065920 . **You have the latest commit that states you've fixed this although the commit does not have any changes. please kindly make sure to make this change and request a third review. Thanks.